README.md

```diff
@@ -27,26 +27,66 @@ export LIBCLANG_PATH=/usr/lib/clang/4.0/lib
 ```
 
 ### Windows
-
 1. Download and unzip [MozTools 4.0](https://github.com/servo/servo-build-deps/releases/download/msvc-deps/moztools-4.0.zip).
 
-2. Download and install Clang for Windows (64 bit) from <https://releases.llvm.org/download.html>.
-
-3. Open up a shell configured to use Visual Studio. This could be the
-   one included with Visual Studio (e.g. Visual Studio 2017 / X64 Native
-   Tools Command Prompt for VS 2017) or a shell in which you have run:
+2. Download and install Clang for Windows (64 bit) from <https://releases.llvm.org/download.html>. 
+   > Note: LLVM under 14 is not able to build mozjs-sys.
 
-   ```shell
-   "c:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Auxiliary\Build\vcvars64.bat"
+3. Download and install `Visual Studio 2022` with `C++ desktop development` with following features:
+   - Windows 11 SDK
+   - ATL
+   - MFC
+  
+    If you want to build in command line, you can download [vs_buildtools.exe](https://aka.ms/vs/17/release/vs_buildtools.exe) linked from and run the following command:
+   ```
+   vs_BuildTools.exe^
+      --add Microsoft.Component.MSBuild^
+      --add Microsoft.VisualStudio.Component.CoreBuildTools^
+      --add Microsoft.VisualStudio.Workload.MSBuildTools^
+      --add Microsoft.VisualStudio.Component.Windows11SDK^
+      --add Microsoft.VisualStudio.Component.VC.CoreBuildTools^
+      --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64^
+      --add Microsoft.VisualStudio.Component.VC.Redist.14.Latest^
+      --add Microsoft.VisualStudio.Component.VC.ATL^
+      --add Microsoft.VisualStudio.Component.VC.ATLMFC^
+      --add Microsoft.VisualStudio.Component.VC.CoreIde^
+      --add Microsoft.VisualStudio.ComponentGroup.NativeDesktop.Core^
+      --add Microsoft.VisualStudio.Workload.VCTools
    ```
 
-4. Set the following environment variables according to where you installed
-   the dependencies above:
+1. Install Python3 and add into PATH.
```

Comment:
```suggestion
3. Install [Python 3.11](https://www.python.org/downloads/windows/) or greater.
  - Ensure that Python is added to the system `PATH`
  - Ensure that a `PYTHON` and `PYTHON3` environment variable point to the Python binary (ie `C:\Python311\python.exe`
```

Reply:
I think mozilla does not always support the latest IIRC, but they definitely support older (python 3.8): https://firefox-source-docs.mozilla.org/build/buildsystem/python.html

Reply:
I've modified this to say 3.11.

Reply:
All I am saying is that I would rather lower python version recommendation, preferably to 3.8, as it is in mozilla.

Reply:
I mean "Python 3.8 or greater" that includes 3.11; it's just to make sure that others do not get wrong impression of minimal supported python version.